src/fhirConfig.ts

```diff
@@ -63,4 +63,5 @@ export interface FhirConfig {
     server: Server;
     logging: Logging;
     profile: Profile;
+    defaultRetryRequestInSeconds: number;

```

Comment:
Just to link the two discussions: https://github.com/awslabs/fhir-works-on-aws-deployment/pull/97#discussion_r485294464

Reply:
fixed